.gitignore

```diff
@@ -7,4 +7,5 @@ Makefile.coq.*
 .depend
 .lia.cache
 .nia.cache
+.DS_Store
```

Comment:
this seems to be a [mac thing](https://en.wikipedia.org/wiki/.DS_Store). I am guessing that you want to add it to your global git config instead. [see here](https://stackoverflow.com/questions/5724455/can-i-make-a-user-specific-gitignore-file) for an example.